src/cli.rs

```diff
@@ -210,6 +210,42 @@ pub struct Opt {
     /// or PAGER (BAT_PAGER has priority).
     #[structopt(long = "paging", default_value = "auto")]
     pub paging_mode: String,
+
+    /// Display line numbers next to the diff. The first column contains line
+    /// numbers in the previous version of the file, and the second column contains
+    /// line number in the new version of the file. A blank cell in the first or
+    /// second column indicates that the line does not exist in that file (it was
+    /// added or removed, respectively).
+    #[structopt(short = "n", long = "number")]
+    pub show_line_numbers: bool,
+
+    /// Color for the left (minus) column of line numbers (--number), if --number is set.
+    /// Defaults to --hunk-color.
+    #[structopt(long = "number-minus-foreground-color")]
+    pub number_minus_foreground_color: Option<String>,
+
+    /// Color for the right (plus) column of line numbers (--number), if --number is set.
+    /// Defaults to --hunk-color.
+    #[structopt(long = "number-plus-foreground-color")]
+    pub number_plus_foreground_color: Option<String>,
```

Comment:
In the new world of the `style-strings` branch I think we're going to want these options to be named `--number-minus-style` and `--number-plus-style`. (It might help to take a look at the new style documentation in `delta --help`.)